extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking/index.js

```diff
@@ -52,25 +53,32 @@ function PanelMeasurementTableTracking({ servicesManager, extensionManager }) {
     debouncedMeasurementChangeTimestamp,
   ]);
 
-  // ~~ DisplayStudySummary
-  useEffect(() => {
+  const test = async () => {
     if (trackedMeasurements.matches('tracking')) {
       const StudyInstanceUID = trackedStudy;
       const studyMeta = DicomMetadataStore.getStudy(StudyInstanceUID);
       const instanceMeta = studyMeta.series[0].instances[0];
-      const { Modality, StudyDate, StudyDescription } = instanceMeta;
+      const { StudyDate, StudyDescription } = instanceMeta;
+
+      const dataSource = extensionManager.getDataSources('dicomweb')[0];
+      const [getStudyResult] = await dataSource.query.studies.search({ studyInstanceUid: StudyInstanceUID });
 
       if (displayStudySummary.key !== StudyInstanceUID) {
         setDisplayStudySummary({
           key: StudyInstanceUID,
           date: StudyDate, // TODO: Format: '07-Sep-2010'
-          modality: Modality,
```

Comment:
This issue was brought up because a study we would normally consider a `CT` was showing as an `SR`. It's possible our image archive provides the most recent image's `modality` as the Study's `modality`, causing the issue. No need to do an async query here. Also, this information _should_ be available in `modalitiesInStudy` which is an odd summary field. Not sure if it's available in instanceMeta

CC: @JamesAPetts

Reply:
Should i wait for @JamesAPetts to add it there and update here?

Reply:
const StudyInstanceUID = trackedStudy;
// forEach trackedSeries (is an array)
DicomMetadataStore.getSeries(StudyInstanceUID, SeriesInstanceUID); 
// get first instance
// add modality to array if not already in array
// string.join('/')
// display